pkgdown/_pkgdown.yml

```diff
@@ -0,0 +1,190 @@
+url: https://www.danieldsjoberg.com/gtsummary/
+
+template:
+  params:
+    docsearch:
+      api_key: fd7c21cad87500b8877c475ed3a4cece
+      index_name: gtsummary
+    ganalytics: UA-145876596-1
+    bootswatch: flatly
+development:
+  mode: auto
+  version_label: default
+
+authors:
+  Daniel D. Sjoberg:
+    href: "https://www.danieldsjoberg.com/"
+  Emily C. Zabor:
+    href: "http://www.emilyzabor.com/"
+  Jessica Lavery:
+    href: "http://www.jessicalavery.com/"
+  Joseph Larmarange:
+    href: "https://joseph.larmarange.net/"
+
+navbar:
+  type: default
+  left:
+  - icon: fa-home
+    href: index.html
+  - text: Reference
+    href: reference/index.html
+  - text: Articles
+    menu:
+    # - text: "tbl_summary() tutorial"
+    #   href: articles/tbl_summary.html
+    # - text: "tbl_regression() tutorial"
+    #   href: articles/tbl_regression.html
+    # - text: "inline_text() tutorial"
+    #   href: articles/inline_text.html
+    # - text: "gtsummary + Quarto/Rmarkdown"
+    #   href: articles/rmarkdown.html
+    # - text: "gtsummary + Shiny"
+    #   href: articles/shiny.html
+    # - text: "gtsummary + themes"
+    #   href: articles/themes.html
+    - text: "cheat sheet"
```

Comment:
I think there is a typo here of one space. Probably not the culprit though @ddsjoberg